cluster/examples/kubernetes/ceph/common.yaml

```
@@ -1712,3 +1712,33 @@ roleRef:
   name: rbd-external-provisioner-runner
   apiGroup: rbac.authorization.k8s.io
 # OLM: END CSI RBD CLUSTER ROLEBINDING
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-admission-controller-sa
+  namespace: rook-ceph
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-admission-controller-role
+rules:
+  - apiGroups: ["ceph.rook.io"]
+    resources: ["cephclusters"]
```

We will want to add validation for all CRDs, right?
```suggestion
    resources: ["*"]
```

---

Added